005_mark_safe_L8/005_mark_safe_L8_safe.py

```
@@ -0,0 +1,43 @@
+    template = loader.get_template('contents.html')
+    # ok:avoid-mark-safe
+    fine = mark_safe(
+        """
+        <div>
+            <p>Content!</p>
+        </div>
+        """
+    )
+    return HttpResponse(template.render({"html_example": fine}, request))
+
+def not_really_safe(request):
```

Ruff (0.14.1)

Redefinition of unused `not_really_safe` from line 7

(F811)